bokehjs/test/unit/models/widgets/tables/cell_formatters.ts

```diff
@@ -122,6 +122,23 @@ describe("cell_formatters module", () => {
         const df = new ScientificFormatter({nan_format: "-"})
         expect(df.doFormat(0, 0, NaN, {}, {})).to.be.equal('<div class="bk" style="text-align: left;">-</div>')
       })
+
+      it("should apply to_fixed if value is zero", () => {
+        const df = new ScientificFormatter({precision: 0})
+        expect(df.doFormat(0, 0, 0, {}, {})).to.be.equal('<div class="bk" style="text-align: left;">0</div>')
+      })
+
+      it("should apply to_fixed if value is 10", () => {
+        const df = new ScientificFormatter({precision: 10})
+        expect(df.doFormat(0, 0, 10, {}, {})).to.be.equal('<div class="bk" style="text-align: left;">10</div>')
+      })
+
+      it("should apply to_fixed if value is 1e-8", () => {
+        const df = new ScientificFormatter({precision: 1e-8})
+        expect(df.doFormat(0, 0, 1e-8, {}, {})).to.be.equal('<div class="bk" style="text-align: left;">1.0e-8</div>')
+      })
+
+
```

Comment:
@kir0ul, can you remove those empty lines for the sake of consistency with the rest of bokehjs?

Reply:
I fixed it in my fork but I didn't see it had already been merged :grimacing: 

Do you use any automatic formatter for this kind of things, like [Prettier](https://prettier.io/) for example?

Reply:
We are still weighing options for automatic code formatting as a team. Sorry I missed this before merging!